wbooks/src/app/index.tsx

```diff
@@ -12,18 +12,35 @@ import React from 'react';
 import { createStackNavigator } from '@react-navigation/stack';
 import { NavigationContainer } from '@react-navigation/native';
 import { RootStackParamList } from '@constants/types';
+import Colors from '@constants/colors';
+import { Image } from 'react-native';
 
 import Library from './screens/Library';
 import BookDetail from './screens/BookDetail';
+import styles from './style';
 
 const Stack = createStackNavigator<RootStackParamList>();
 
+const HeaderBackground = () => (
+  <Image source={require('@assets/General/bc_nav_bar.png')} resizeMode="contain" style={styles.header} />
+);
```

Comment:
No, create a generic component in `src/app/components` for this. 
Also, we don't use `require` for imports:

```
import headerBg from '@assets/General/bc_nav_bar.png';
...
<Image source={headerBg} resizeMode="contain" style={styles.header} />
```